callmetadata.schema.json

```diff
@@ -7,8 +7,37 @@
     "definitions": {
         "participant": {
             "type": "object",
-            "description": "Specific details about the farEndUsers (the other end of the call)",
+            "description": "Specific details about a participant of the call",
+            "additionalProperties": true,
             "properties": {
+                "name": {
+                    "type": "string",
+                    "example": "Steve Jones",
+                    "description": "Full name of the participant on the call"
+                },
+                "address": {
+                    "type": "string",
+                    "example": "sip:9991234567@voip-provider.net",
+                    "description": "The address of the participant"
+                },
+                "startTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "example": "2016-06-18T07:50:30Z",
+                    "description": "Date and time when the participant was requested to participate on the call"
+                },
+                "joinTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "example": "2016-06-18T07:50:30Z",
+                    "description": "Date and time when the participant joined the call"
+                },
+                "endTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "example": "2016-06-18T08:15:15Z",
+                    "description": "Date and time when the participant leaved the call"
```

Comment:
`leaved` is not the best English grammar.  `left` is more correct, but `exited` might be better.
irregulars are hard.

Reply:
@tschady Thank you. Yes, irregulars are hard. Fixed.